TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -397,6 +397,10 @@ TYPO3:
             authenticationProviderName: '${entity.authenticationProviderName}'
             name: '${entity.party.name.fullName}'
 
+    login:
+      styles:
+        'resource://TYPO3.Neos/Public/Styles/Login.css': TRUE
```

Comment:
This creates a coupling to the file path/name. It's probably not really an issue in practice, but if the path changes for some reason, a different setting that overwrites it would be appended instead of replaced.
Again, more a theoretical problem, but something like this could work as well:
```
'TYPO3.Neos:DefaultStyles': 'resource://TYPO3.Neos/Public/Styles/Login.css'
```